lib/mailkick/service/mandrill.rb

```diff
@@ -26,8 +29,10 @@ def opt_outs
         end
       end
 
+      # TODO remove ENV["MANDRILL_APIKEY"]
       def self.discoverable?
-        !!(defined?(::Mandrill::API) && ENV["MANDRILL_APIKEY"])
+        !!(defined?(::Mandrill::API) &&
+          ENV["MANDRILL_API_KEY"] || ENV["MANDRILL_APIKEY"]
```

Comment:
Isn't `(true && false || true) == true`, so wouldn't the above work? But for readability reasons, :+1: